### PR TITLE
Release '0.14.0'

### PR DESCRIPTION
**Pull request by evidolob**

### CODE REVIEW

**Review by JPinkney — APPROVED** [review]
LGTM. Any chance you can take a look at https://github.com/redhat-developer/vscode-yaml/pull/405 and maybe we can merge it in. If not, we can just add it into the next release

**Comment by evidolob** [comment]
@JPinkney  I already approve #405, going to read it again...

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+### 0.14.0
+
+- yaml-language-server use a non-standard LSP request to resolve schemas content on client [#359](https://github.com/redhat-developer/yaml-language-server/pull/359)
+- Add in non-standard lsp request for resolving schemas on the client-side [#395](https://github.com/redhat-developer/vscode-yaml/pull/395)
+- Fix error on completion 'null' value [#360](https://github.com/redhat-developer/yaml-language-server/pull/360)
+- Select schemas based off on their priority [#362](https://github.com/redhat-developer/yaml-language-server/pull/362)
```
Comment (joshuawilson) [inline CHANGELOG.md:6]: Is this a typo? `based off on their` "on" should be "of"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+- Update readme with example of an array of glob patterns for schema [#366](https://github.com/redhat-developer/yaml-language-server/pull/366)
+- Add Dockerfile [#335](https://github.com/redhat-developer/yaml-language-server/issues/335)
+- Fix: Code completion list empty on empty file [#349](https://github.com/redhat-developer/vscode-yaml/issues/349)
+- Fix: Autocompletion missing space in value for default snippets when autocompleting on root node [#364](https://github.com/redhat-developer/yaml-language-server/issues/364)
+- Check if dynamic registration is enabled before executing onDidChangeWorkspaceFolders [#378](https://github.com/redhat-developer/yaml-language-server/pull/378)
+- Fix: Array indentation in autocomplete is broken after upgrade to 0.13 [#376](https://github.com/redhat-developer/yaml-language-server/issues/376)
+- Added folding ranges provider implementation [#337](https://github.com/redhat-developer/yaml-language-server/issues/337)
+- Fix: Hover doesn't work when there is now symbol after property [#382](https://github.com/redhat-developer/yaml-language-server/pull/382)
```
Comment (joshuawilson) [inline CHANGELOG.md:15]: Typo: `now symbol` -> `no symbol`

**Comment by joshuawilson** [comment]
What happened to the `package-lock.json` file? It has a LOT of changes.

**Comment by evidolob** [comment]
@joshuawilson I use npm 7.x to generate `package-lock.json`, in 7.x thay updated `package-lock.json` `lockfileVersion` to `2`, I assume that new version have a different structure, any way I revert that.

**Review by joshuawilson — APPROVED** [review]

**Pull request merged**